assembly/std/index.ts

```diff
@@ -316,10 +316,31 @@ export function unsafeRandom(): i64 {
 /**
  * Sends an async message to a function at given address.
  *
+ * @remarks
+ * The goal of sendMessage functionality is to send a message in the future, that will be executed as soon as possible
+ * after the start period but not after the end period. This message allows you to make executions in the future and
```

Comment:
After the first sentence: "The goal of sendMessage functionality is to send a message in the future, that will be executed as soon as possible after the start period but not after the end period." include:
 
You might want to use the sendMessage functionality:

- Having a smart contract called periodically, without a centralized bot;
- Having a smart contract that will trigger on the change of value (for example a change in price), of an other one;
- Having an object that evolves on the blockchain itself.

Continue with the rest of the content "This message allows you".....